### PR TITLE
[15184] Move one item between holdings within an instance (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by dnovikau**
[C15184](https://foliotest.testrail.io/index.php?/cases/view/15184)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5083/allure)
***

### CODE REVIEW

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ describe('Orders: Inventory interaction', () => {
               testData.order = order;
```
Comment (TetianaParanich) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:49]: please fix linter issue

File: cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ describe('Orders: Inventory interaction', () => {
               testData.orderNumber = order.poNumber;
```
Comment (TetianaParanich) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js:71]: please fix linter issue

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 3]
```diff
@@ -903,16 +920,19 @@ export default {
...
+      cy.do(holdingsSection.clickHeader());
+    }
 
     records.forEach((record, index) => {
       if (record.barcode) {
         cy.expect(
           holdingsSection
             .find(MultiColumnListRow({ rowIndexInParent: `row-${index}` }))
-            .find(MultiColumnListCell({ columnIndex: 0 }))
+            .find(MultiColumnListCell({ columnIndex }))
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/inventoryInstance.js:935]: Why you delete index?

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:49]: Done

File: cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js (hunk 2 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js:71]: Done

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 3 above)
Comment (dnovikau) [reply cypress/support/fragments/inventory/inventoryInstance.js:935]: Right now it's passed as parameter to the function, it done for cases when 'move item' operation is active and barcode column shifted 2 columns right

Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryInstance.js:935]: thanks

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/inventory/moving-item-between-holdings.cy.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+          cy.login(testData.user.username, testData.user.password, {
+            path: TopMenu.inventoryPath,
+            waiter: InventoryInstances.waitContentLoading,
+          });
+        },
+      );
+    });
+
+    after('Delete test data', () => {
```
Comment (manowarqqq) [inline cypress/e2e/inventory/moving-item-between-holdings.cy.js:67]: Need to remove Location

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**